### ISSUE TITLE
Sinric.com seems to be down ...

### ISSUE THREAD

**Issue by MacSass** [kakopappa/sinric#341]
Hi @kakopappa , 
looks like sinric.com is down - my devices don´t work anymore and sinric.com only returns a cloudflare error 522 (timeout).

Hope it will be back soon ...

**Comment by amitabh-srivastav** [comment]
Yes, it appears that the server is down. Dear @kakopappa please attend to It ASAP.

**Comment by kakopappa** [comment]
Looks like VPS is down. opened a support ticket.

**Comment by kakopappa** [comment]
Up

**Comment by MacSass** [comment]
Great - confirmed - thank you!

**Comment by amitabh-srivastav** [comment]
Thanks
